en/04_Changelogs/beta/5.0.0-beta1.md

```diff
@@ -11,7 +72,81 @@ We will only consider merging post-beta breaking API changes if they are necessa
+
+### Planning your upgrade
+
+You should decide if you intend to do a new major release of your module to add support for Silverstripe CMS 5.
+
+Most of the key Silverstripe CMS APIs have not changed between CMS 4 and CMS 5. That said, take time to read this changelog to identify changes that might be problematic for your project.
+
+Depending on how complex your module is, it may be completely practical to create a version that works with both Silverstripe CMS 4 and Silverstripe CMS 5. We do not necessarily recommend this approach. It may require you to tag a major release later on if you want to make use of new SilverstripeCMS 5 feature later on. But you may wish to consider this approach to minimise your maintenance burden.
+
+If you decide that you want to maintain distinct Silverstripe CMS 4 and Silverstripe CMS 5 versions of your module, then you should create a new branch from your current development branch.
+
+[info]
+If the name of your branch is a number, composer will allow it to be installed as an unstable tag. e.g.: If your module has a `7` branch, composer will alias the `HEAD` of that branch as `7.x-dev`. This will satisfy a constraint like `^7`. This is especially useful for some development pipelines and for running tests in a continuous integration environment like Github Actions.
+[/info]
+
+### Stop using deprecated code
+
+A good way to identify which APIs need to be updated is to [enable deprecation warnings](/Upgrading/Deprecations) and run your module in a fresh project using the latest Silverstripe CMS 4 release. This will let you remove any references to deprecated APIs in the Silverstripe CMS 4 version of your module.
+
+If you intend to remove some APIs in the Silverstripe CMS 5 version of your module, consider [adding deprecation warnings](/contributing/release_process#deprecating-api) to the Silverstripe CMS 4 version. This will simplify the upgrade process for projects using your module.
+
+### Upgrade your constraints
+
+Upgrade your composer requirement constraints to allow the installation of Silverstripe CMS 5 module. If your module has dual support for Silverstripe CMS 4 and 5, you'll want to use a constraint similar to:
+```json
+"require": {
+    "silverstripe/framework": "^4 || ^5",
+    "silverstripe/cms": "^4 || ^5",
+    "silverstripe/admin": "^1 || ^2"
+}
+```
+
+Otherwise, simply increment your existing constraints to target the Silverstripe CMS 5 version of our modules.
+
+### Ship a Silverstripe CMS 5 version of your module
+
+Obviously, you'll want to test your module against a Silverstripe CMS 5 installation to identify and fix any problems. Once you're confident that your module works well with Silverstripe CMS 5, you'll want to tag a new release.
+
+Consider tagging a beta release if you want to give your users a chance to test drive your module with the Silverstripe CMS 5 beta.
```

Comment:
```suggestion
```
If you take my suggestion above, we can remove this paragraph.

Reply:
Fixed